uplink/ecclient/client.go

```diff
@@ -271,14 +271,22 @@ func (ec *ecClient) putPiece(ctx, parent context.Context, limit *pb.AddressedOrd
 		Address: limit.GetStorageNodeAddress(),
 	})
 	if err != nil {
-		ec.log.Sugar().Debugf("Failed dialing for putting piece %s to node %s: %v", pieceID, storageNodeID, err)
+		ec.log.Debug("Failed dialing for putting piece to node",
+			zap.Binary("pieceID", pieceID.Bytes()),
```

Comment:
any reason to do this instead of using the string version (`pieceID.String()`)?

Reply:
I asked yesterday in our [slack engineering channel](https://storj.slack.com/archives/CAQV0AE2Z/p1564426226278400) regarding UTF-8 and based on the responses I thought that binary was better.

Reply:
It should be completely fine to use `zap.String("pieceID", pieceID)`, all the primitive types have an encoding that is ascii safe.

Reply:
I'll swap it over, should I do the same with `nodeID`?

Reply:
Yeah.